### PR TITLE
 Refactor UnknownEntityException to be localized and coded

### PR DESCRIPTION
**Pull request by dennishendriksen**
- Fix incorrect references to UnknownEntityException with UnknownEntityTypeException.

#### Checklist
- [ ] Functionality works & meets specifications
- [ ] Code reviewed
- [ ] Code unit/integration/system tested
- [ ] User documentation updated
- [ ] (If you have changed REST API interface) view-swagger.ftl updated
- [ ] Test plan template updated
- [ ] Clean commits

### CODE REVIEW

File: molgenis-semantic-mapper/src/main/java/org/molgenis/semanticmapper/controller/TagWizardController.java [hunk 1]
```diff
@@ -95,7 +95,7 @@ public String viewTagWizard(@RequestParam(required = false, value = "selectedTar
 
 		if (StringUtils.isEmpty(target))
 		{
-			throw new UnknownEntityException("There are no entities available!");
+			throw new MolgenisRuntimeException("There are no entities available!");
```
Comment (fdlk) [inline molgenis-semantic-mapper/src/main/java/org/molgenis/semanticmapper/controller/TagWizardController.java]: Should be IllegalStateException.

File: molgenis-semantic-mapper/src/main/java/org/molgenis/semanticmapper/service/impl/MappingServiceImpl.java [hunk 2]
```diff
@@ -93,7 +94,7 @@ public MappingProject cloneMappingProject(String mappingProjectId)
 		MappingProject mappingProject = mappingProjectRepository.getMappingProject(mappingProjectId);
 		if (mappingProject == null)
 		{
-			throw new UnknownEntityException("Mapping project [" + mappingProjectId + "] does not exist");
+			throw new UnknownEntityException(dataService.getEntityType(MAPPING_PROJECT), mappingProjectId);
```
Comment (fdlk) [inline molgenis-semantic-mapper/src/main/java/org/molgenis/semanticmapper/service/impl/MappingServiceImpl.java]: autowire the metadata and use it where needed

File: molgenis-semantic-search/src/main/java/org/molgenis/semanticsearch/service/impl/OntologyTagServiceImpl.java [hunk 3]
```diff
@@ -114,7 +114,7 @@ public void removeAttributeTag(EntityType entityType, SemanticTag<Attribute, Ont
 
 		if (packageEntity == null)
 		{
-			throw new UnknownEntityException("Unknown package [" + package_.getId() + "]");
+			throw new UnknownEntityException(dataService.getEntityType(PACKAGE), package_.getId());
```
Comment (fdlk) [inline molgenis-semantic-search/src/main/java/org/molgenis/semanticsearch/service/impl/OntologyTagServiceImpl.java]: Autowire the metadata

**Comment by fdlk** [comment]
See also #7098

**Comment by dennishendriksen** [comment]
Close for now, should be reopened in the near future.

**Pull request closed without merging**